### DISCUSSION TITLE
OpenAI: {"detail":"Our systems have detected unusual activity from your system. Please try again later."} (code: 403)

### DISCUSSION THREAD

**Discussion by luluhuhu** [acheong08/ChatGPT#1421] [Q&A]
revChatGPT V1 reported an error: OpenAI: {"detail":"Our systems have detected unusual activity from your system. Please try again later."} (code: 403). May I ask what is the reason?

**Comment by ggg5438** [comment]
Hi see you again 😁

I got the same error.
